### ISSUE TITLE
Alternative grid paging mode

### ISSUE THREAD

**Issue by slehoux** [ag-grid/ag-grid#2210]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[X] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
Right now it would seem that the only way to display the paginf is with a system of "Previous" and "Next" buttons when implementing paging in the grid. 

**Expected behavior**
Do you have plans to implement a "Page List", i.e. a list of pages you can click. If it is already implemented please direct me to the right information. 

Sorry in advance if the question has already been answered.

* **ag-Grid version:** 14.2.0

* **Language:**  TypeScript 2.3.0

**Comment by Koen967** [comment]
I don't believe there is currently a feature that provides that out of the box. But you could create your own page list using the paginationGetTotalPages() method to get the number of pages. Put all the values between 0 and the total pages in a dropdown box and call paginationGoToPage(pageNumber) on selecting a value from that dropdown box.
For more information about paging methods look [here.](https://www.ag-grid.com/javascript-grid-pagination/#pagination-api)

**Comment by makinggoodsoftware** [comment]
Hi

Thanks @Koen967 that is right

To give more context, you can create your own pagination controls as showcased here:

https://www.ag-grid.com/javascript-grid-pagination/#custom-pagination-controls

Note that we are going to flag this thread as managed by the community
